qiskit/transpiler/synthesis/permrowcol.py

```diff
@@ -27,13 +27,17 @@
 from qiskit.circuit.library.generalized_gates.permutation import Permutation
 from qiskit.circuit.exceptions import CircuitError
 
+from qiskit.synthesis.linear.linear_circuits_utils import calc_inverse_matrix
+
 
 class PermRowCol:
     """Permrowcol algorithm"""
 
     def __init__(self, coupling_map: CouplingMap):
         self._coupling_map = coupling_map
         self._graph = pydigraph_to_pygraph(self._coupling_map.graph)
+        self.choose_column_function = self.choose_column
```

Comment:
I assume this is so that the heuristics of choosing row & column can be changed? If so, good addition I think.